src/components/partnershipCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { Partnership } from "@prisma/client";
+import Link from "next/link";
+import { Twitter } from "lucide-react";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "~/components/ui/card";
+
+import { Badge } from '~/components/ui/badge';
+import { Separator } from "~/components/ui/separator";
+
+import { cn } from "~/lib/utils"
+
+type PartnershipCardProps = React.HTMLAttributes<HTMLDivElement> & {
+  partnership: Partnership;
+}
+
+function TwitterButton({ uri }: { uri: string }) {
+  return (
+    <a href={uri} target="_blank" className="rounded-full bg-gray-100 p-2 text-black">
+      <Twitter size={16} />
+    </a>
+  );
+}
```

Comment:
Let's extract it to a separate file as we gonna reuse it on Proposal details page